customers/css/stylecustomers.css

```diff
@@ -0,0 +1,101 @@
+
+
+.customers__body {
+	padding: 0 15px;
+	margin: 0 auto;
+	font-family: 'Raleway';
+	width: 1170px;
+}
+
+.customers__title {
+	font-family: 'Raleway';
+	font-size: 36px;
+	line-height: 42px;
+	text-transform: uppercase;
+	color: #000000;
+	font-size: 36px;
+	margin: 60px 0 50px 0;
+	text-align: center;
+}
+
+.customers__line {
+	width: 170px;
+	height: 0;
+	border: 1px solid #EC7979;
+	margin: 60px auto 30px;
+}
+
+.customers__subtitle {
+	width: 1170px;
+	font-size: 18px;
+	line-height: 21px;
+	color: #86878B;
+	text-align: center;
+}
+
+.customers__row {
+	margin-top: 83px;
+	display: flex;
+	justify-content: space-between;
+	flex: 0 1 360px;
+}
+
+
+.customers__text {
+	background: #E2E2E2;
+	border-radius: 10px;
+	width: 360px;
+	height: 180px;
+	text-align: center;
+	color: #86878B;
+	padding: 38px 20px;
+	font-size: 18px;
+}
+
+.customers__text::after{
+	content: "";
+	width: 40px;
+	height: 30px;
+	border-left: 15px solid transparent;
+	border-right: 15px solid transparent;
+	border-top: 20px solid #E2E2E2;
+	position: relative;
+	top: 90px;
+	right: 150px;
+}
+
+.customers__profeli {
+	margin-top: 66px;
+	margin-bottom: 64px;
+	width: 370px;
+	display: flex;
+	align-items: flex-start;
+}
+
+.customers__profeli-photo {
+	display: inline-flex;
+	width: 70px;
+	height: 70px;
+}
+
+.customers__profeli-name{
+	margin-left: 30px;
+}
+
+.name {
+	margin-top: 0;
+	padding-top: 15px;
+	font-size: 24px;
+	line-height: 33px;
+	text-transform: capitalize;
+	color: #000000;
+	font-family: 'Open Sans';
+	height: 54px;
+}
+
+.manager {
+	margin-top: -15px;
```

Comment:
Please, replace a negative margin (add margin or padding to someone else to do not use a negative margins)